src/project/mod.rs

```diff
@@ -164,6 +164,20 @@ impl Project {
                 PYPROJECT_MANIFEST
             ),
         };
```

Comment:
This function bails if there is no manifest available. So it will never reach the next if statement. I'm not sure if this happens because of the main merge but I can't get this pr to work :sweat_smile:

Reply:
🤦 I hadn't tested this outside of any directories with manifests since refactoring to work in `discover()`.